components/ActivityCard/index.js

```diff
@@ -21,6 +21,32 @@ export default function ActivityCard({ name, date, time, joined, category }) {
     }
   }, [name]);
 
+  // check date to display date, weekday, "today" or "tomorrow" //////////////////////////////
```

Comment:
I like the idea of commenting functions etc. ^^

Reply:
same